summary/table.go

```diff
@@ -136,3 +134,27 @@ func WriteTable(header []string, data [][]string) {
 	}
 	table.Render()
 }
+
+func writePodInfoToTable(podname, namespace, clustername, containername, labels string) {
+
+	fmt.Printf("\n")
+
+	podinfo := [][]string{
+		{"Pod Name", podname},
+		{"Namespace Name", namespace},
+		{"Cluster Name", clustername},
+		{"Container Name", containername},
+		{"Labels", labels},
+	}
+	table := tablewriter.NewWriter(os.Stdout)
```

Comment:
We have tablewriter function in probe and recommend as well. Can we check if the following code as refactored?

Reply:
Hello @daemon1024 
This file also contains a tableWriter function. But since I wanted to write this info with no border, I wrote a new function.